yoyodyne/train.py

```diff
@@ -296,10 +309,10 @@ def get_model(
         encoder_layers=encoder_layers,
         end_idx=train_set.index.end_idx,
         expert=expert,
-        features_vocab_size=getattr(
-            train_set.index, "features_vocab_size", -1
-        ),
-        features_idx=getattr(train_set.index, "features_idx", -1),
+        features_vocab_size=features_vocab_size,
+        features_idx=train_set.index.source_vocab_size
```

Comment:
Is this only here for the feature invariant transformer now? I think we could make the same change to that module. The features_idx is only used to zero out positional embeddings for features, since their positional info is irrelevant.